scripts/linters/js_ts_linter.py

```diff
@@ -960,6 +971,74 @@ def _check_comments(self):
         return concurrent_task_utils.TaskResult(
             name, failed, error_messages, error_messages)
 
+    def _check_angular_services_index(self):
+        """Finds all @Injectable classes and makes sure that they are added to
+            Oppia root and Angular Services Index.
+
+        Returns:
+            all_messages: str. All the messages returned by the lint checks.
+        """
+        name = 'Angular Services Index file'
+        error_messages = []
+        def get_injectable_class_name(file_content):
+            """Extarcts the class name from a file that has an Injectable
+                class.
```

Comment:
Deindent by 4

Reply:
The comment doesn't exist anymore as the function was removed.